src/app/rules/rules-definition/changeTurn.ts

```diff
@@ -22,13 +22,9 @@ export class ChangeTurn extends BaseRules {
   public run(args?: any): boolean {
     if (this._refObj.threeDModel.oldPosition[0] === -1 && this._refObj.threeDModel.oldPosition[1] === -1)
       return true;
-    if ((this._refScene.player === 0 || this._refScene.player === 1) && this._refObj.name === 'whitepawn')
-      this._refScene.player += 1;
-    else if ((this._refScene.player === 2 || this._refScene.player === 3) && this._refObj.name === 'blackpawn') {
-      this._refScene.player += 1;
-      if (this._refScene.player === 4)
-        this._refScene.player = 0;
-    } else {
+    if ((this._refScene.player === 0 || this._refScene.player === 2) && this._refObj.name === 'whitepawn') {}
+    else if ((this._refScene.player === 1) && this._refObj.name === 'blackpawn') {}
+    else {
       return false;
```

Comment:
tu n'as pas moyen de virer les 2 conditions du dessus ? ou alors de les retravailler ?

Reply:
if (this._refObj.threeDModel.oldPosition[0] === -1 && this._refObj.threeDModel.oldPosition[1] === -1)

Celle ci ?

Reply:
```
if (![0, 1, 2].include(this._refScene.player) || !['whitepawn', 'blackpawn'].include(this._refObj.name))
	return false
```

Reply:
Ok vu.

J'avais laissé comme ça au cas ou on est des actions spécial en fonctions des joueurs.